src/shared/util/gvector.h

```diff
@@ -212,9 +212,9 @@ vector3d<num> cross(const vector3d<num> a,const vector3d<num> b)
   template <class num> \
   vector3d<num> &vector3d<num>::operator opr (const vector3d<num> p) \
   {                  \
-    x = x opr p.x;   \
```

Comment:
I think it was g++ that complained about these assignments. Took me a while to understand what's happening there.